### PR TITLE
Add Support For ADA Addresses In Encoding Library

### PR DESCRIPTION
**Pull request by twenty1Pi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ This library currently supports the following cryptocurrencies and address forma
  - NEM (base32)
  - EOS
  - KSM (ss58)
+ - ADA (bech32)
```
Comment (makoto) [inline README.md:49]: Can you provide reference link so that we can verify?

Comment (twenty1Pi) [reply README.md:49]: https://forum.cardano.org/t/cip5-common-bech32-prefixes/35189
^^ @makoto

File: src/__tests__/index.test.ts [hunk 2]
```diff
@@ -206,6 +206,13 @@ const vectors: Array<TestVector> = [
     passingVectors: [
       { text: 'cosmos1depk54cuajgkzea6zpgkq36tnjwdzv4afc3d27', hex: '6e436a571cec916167ba105160474b9c9cd132bd' }
     ],
+  },
+  {
+    name: 'ADA',
+    coinType: 1815,
+    passingVectors: [
+      { text: 'addr1gqtnpvdhqrtpd4g424fcaq7k0ufuzyadt7djygf8qdyzevuph3wczvf2dwyx5u', hex: '401730b1b700d616d51555538e83d67f13c113ad5f9b22212703482cb381bc5d81312a' }
+    ],
```
Comment (makoto) [inline src/__tests__/index.test.ts:215]: It seems that ADA has `addr` prefix for mainnet and `addr_test` for testnet 
https://github.com/cardano-foundation/CIPs/pull/6/files#diff-50134d380e035316e17c63951fe2d813R33

@Arachnid do we enforce the support of test address as well?

Comment (twenty1Pi) [reply src/__tests__/index.test.ts:215]: @makoto Do we want the support for test address as well?

**Review by Arachnid — APPROVED** [review]

**Pull request merged**